sdk/communication/communication-chat/test/chatClient.spec.ts

```diff
@@ -16,53 +18,247 @@ describe("ChatClient", function() {
   let testUser: CommunicationIdentifier;
   let testUser2: CommunicationIdentifier;
 
-  this.afterAll(async function() {
+  this.afterAll(async function () {
```

Comment:
There are a lot of redundant spaces in the change. Could you please run `rushx format` and sent a new iteration?